src/decision/src/#gps_decision.cpp#

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
I don't think this file should exist, looking at the file name it looks like a temp file.